src/main/java/it/gov/pagopa/pu/debtpositions/model/view/receipt/ReceiptCustom.java

```diff
@@ -0,0 +1,31 @@
+package it.gov.pagopa.pu.debtpositions.model.view.receipt;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+import java.time.OffsetDateTime;
+
+@Entity
+@Table(name = "receipt")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode(callSuper = false)
+public class ReceiptCustom implements Serializable {
```

Comment:
ReceiptCustom is a name ambigous
```suggestion
public class ReceiptView implements Serializable {
```